integration/airflow/tests/failures/airflow/dags/wait_dag.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.python_operator import PythonOperator
+from airflow.utils.dates import days_ago
+from openlineage.client import set_producer
+
+
+_PRODUCER="https://github.com/OpenLineage/OpenLineage/tree/0.0.1/integration/airflow"
+
+set_producer(_PRODUCER)
+
+from airflow.version import version as AIRFLOW_VERSION
+from pkg_resources import parse_version
+if parse_version(AIRFLOW_VERSION) < parse_version("2.0.0"):
+    from openlineage.airflow import DAG
+else:
+    from airflow import DAG
+
+
+default_args = {
+    'owner': 'datascience',
+    'depends_on_past': False,
+    'start_date': days_ago(7),
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'email': ['datascience@example.com']
+}
+
+
+dag = DAG(
```

Comment:
Could not spot difference betweem `wait_dag` and `test_dag`.
If there is a reason for that, please provide it in comment.